src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/tasks/ServiceChannelUpdateTask.java

```diff
@@ -34,5 +34,10 @@ public Long getOrderIndex() {
   public void setOrderIndex(Long orderIndex) {
     this.orderIndex = orderIndex;
   }
+
+  @Override
+  public String getUniqueId() {
+    return String.format("ptv-service-channel-update-task-%s", id.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "id" is already a string, there's no need to call "toString()" on it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)